php_fpm/conf.yaml.example

```diff
@@ -24,6 +24,9 @@ instances:
     # pass in a custom Host header like so
     # http_host: such.production.host
     #
+    # If you need to specify a custom timeout in seconds (default is 20):
+    # timeout: 5
```

Comment:
This default timeout is not consistent with the one specified in the python script (20s) - we probably would like the default uncommented value to be consistent with that.